sys/shell/commands/sc_netif.c

```diff
@@ -171,6 +200,14 @@ void _netif_list(kernel_pid_t dev)
         printf(" NID: 0x%" PRIx16 " ", u16);
     }
 
+    res = ng_netapi_get(dev, NETCONF_OPT_STATE, 0, &state, sizeof(state));
+
+    if (res >= 0) {
+        printf(" State: ");
+        _print_netconf_state(state);
+        puts("");
```

Comment:
Don't put `puts("")` here. The `printf()` below will take care of proper newlining